en/guide/extending.md

```diff
@@ -15,39 +15,35 @@ Video Streaming | Developers can implement new types of video streaming (e.g. HL
 
 ## Support Custom Camera Device
 
-In order to support a new type of camera a custom class must be derived from `CameraDevice`. In addition, code must be added to detect the new device type and to instantiate the new custom camera device when this happens. The sections below detail these steps.
+Developers may want to add support for a new type of camera device that is not currently supported by DCM. To support a custom type of camera, a `PluginXXX` class derived from `PluginBase` and `CameraDeviceXXX` class derived from `CameraDevice` must be implemented. An example of a plugin for a custom camera device with classes `PluginCustom`(https://github.com/Dronecode/camera-manger/tree/master/plugins) and `CameraDeviceCustom`(https://github.com/Dronecode/camera-manger/tree/master/plugins) is added in the project and may be referenced.
```

Comment:
The source code links below don't work (I get 404 so possibly secured).

Reply:
Typo. Need to be replaced by this - https://github.com/Dronecode/camera-manager/tree/master/plugins/CustomCamera